code/addons/test/package.json

```diff
@@ -74,19 +74,19 @@
   },
   "devDependencies": {
     "@types/semver": "^7",
-    "@vitest/browser": "^2.0.0",
+    "@vitest/browser": "^2.1.3",
     "boxen": "^8.0.1",
     "find-up": "^7.0.0",
     "picocolors": "^1.1.0",
     "semver": "^7.6.3",
     "tinyrainbow": "^1.2.0",
     "ts-dedent": "^2.2.0",
-    "vitest": "^2.0.0"
+    "vitest": "^2.1.3"
   },
   "peerDependencies": {
-    "@vitest/browser": "^2.0.0",
+    "@vitest/browser": "^2.1.3",
     "storybook": "workspace:^",
-    "vitest": "^2.0.0"
+    "vitest": "^2.1.3"
```

Comment:
Are these peer deps changes necessary? Can't this change still work with Vitest 2.0.0 and onwards?

The only logic change I see in this PR is moving the import to be a dynamic import, and that's probably still compatible with vitest 2.0.0

Reply:
As you say, these updates were unnecessary and I have reverted.

[97f93b7](https://github.com/storybookjs/storybook/pull/29407/commits/97f93b7d714ec32d42e21d3e5f5b6d6a64c9bf03)